tests/test_integrated_channels/test_cornerstone/test_exporters/test_content_metadata.py

```diff
@@ -113,6 +114,50 @@ def test_transform_description(self, item_description, expected_description):
         exporter.LONG_STRING_LIMIT = 100
         assert exporter.transform_description(item_content_metadata) == expected_description
 
+    @ddt.data(
+        (
+            {'key': 'edx+181'},
+            'edx+181',
+        ),
+        (
+            {'key': 'edx+.'},
+            encode_course_key_into_base64('edx+.'),
+        ),
+        (
+            {'key': 'edx+ '},
+            encode_course_key_into_base64('edx+ '),
+        ),
+        (
+            {'key': 'edx+<>'},
+            encode_course_key_into_base64('edx+<>'),
+        ),
+        (
+            {'key': 'edx+&'},
+            encode_course_key_into_base64('edx+&'),
+        ),
+        (
+            {'key': 'edx+%'},
+            encode_course_key_into_base64('edx+%'),
+        ),
+        (
+            {'key': 'edx+|'},
+            encode_course_key_into_base64('edx+|'),
+        ),
+        (
+            {'key': 'edx+/'},
+            encode_course_key_into_base64('edx+/'),
+        ),
+    )
+    @responses.activate
+    @ddt.unpack
+    def test_transform_ke (self, item_key, expected_id):
```

Comment:
Please add a test case where invalid characters appear in the beginning and middle of the keys, not just at the end.